### ISSUE TITLE
Accessibility: Email footer #credit fails color contrast and there's no setting for users to override it

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43603]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Text in the credit in the email footer is set just to lighten the color of the body text. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/emails/email-styles.php#L100C30-L100C46

There is also no selector in admin settings to define the color for text in the credit. Even if the body text is quite dark and the background is set to nearly white, the 40% lighting results in a color contrast failure for text in the credit:
<img width="1087" alt="Screenshot showing color contrast issue in footer of email template" src="https://github.com/woocommerce/woocommerce/assets/13153456/ea6c54fe-f501-4f14-a534-d44489485fda">

The only way to resolve the color contrast issue in this text is to add a span tag in the credit field and hardcode a color in that way.

### Expected behavior

Two things should happen here:

1. The default email template colors should be changed so that the colors pass contrast checks in their default state, if a website owner never changes them. [See this contrast checker for reference.](https://webaim.org/resources/contrastchecker/)
2. There should be a setting in the WP admin email settings allowing users to change the color of the credit text without having to use HTML in the credit text box.

### Actual behavior

There is a color contrast failure that is not easily fixed.

### Steps to reproduce

Open the default WooCommerce email settings and preview the email or reference the code link above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by amesplant** [comment]
I have started working on this.
